test/addons/argocd/test

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+import time
+
+from drenv import kubectl
+from drenv import commands
+
+
+def wait(cluster):
+    print("Waiting until all deployments are available")
+    kubectl.wait(
+        "deploy",
+        "--all",
+        "--for=condition=Available",
+        "--namespace=argo-test",
+        "--timeout=300s",
+        context=cluster,
+    )
```

Comment:
This runs after start, which should finish only when everything is up. Why we need to wait here again?

Reply:
previous step creates an argocd application, and here we need wait for this application to be healthy status.

Reply:
updated code.